src/google/protobuf/descriptor.proto

```diff
@@ -396,7 +396,9 @@ message FileOptions {
 
 
   // Sets the objective c class prefix which is prepended to all objective c
-  // generated classes from this .proto. There is no default.
+  // generated types from this .proto. Note that if the type already has
```

Comment:
Not sure it is worth this tweak pending what the final decision is about prefix == input cases.